app/src/main/java/com/hedvig/app/util/apollo/GraphQLUtils.kt

```diff
@@ -39,3 +39,8 @@ fun MonetaryAmount.format(context: Context, market: Market?, minimumDecimals: In
         it.currency = Currency.getInstance(currency.currencyCode)
         it.minimumFractionDigits = minimumDecimals
     }.format(this.number.numberValueExact(BigDecimal::class.java))
+
+fun MonetaryAmount.getCurrencyString(context: Context, market: Market?): String? =
+    NumberFormat.getCurrencyInstance(getLocale(context, market)).also {
+        it.currency = Currency.getInstance(currency.currencyCode)
+    }.currency?.currencyCode
```

Comment:
Could this function be simplified to `currency.currencyCode`?

Reply:
Also, I don't think we want to display the ISO-4217-code to users. We're most likely going to want to use `Currency::getSymbol(Locale)`. Sample output from some quick `jshell`-testing:

```java
jshell> import java.util.Currency;

jshell> Currency c = Currency.getInstance("SEK");
c ==> SEK

jshell> c
c ==> SEK

jshell> c.getSymbol();
$4 ==> "kr"

jshell> import java.util.Locale;

jshell> c.getSymbol(Locale.forLanguageTag("en-SE"));
$6 ==> "kr"

jshell> c.getSymbol(Locale.forLanguageTag("en-US"));
$7 ==> "SEK"
```

Reply:
Aha! Thanks. Fixed!

Reply:
This was also entirely obsoleted by an update in the design. We should now be able to simply use `MonetaryAmount::format` in order to populate the view with the price, and have the per month-label be static 😄